crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -2,12 +2,15 @@
 //! This module contains functions that are used to verify blocks and transactions, and doesn't
 //! assume anything about the chainstate, so it can be used in any context.
 //! We use this to avoid code reuse among the different implementations of the chainstate.
-
 extern crate alloc;
+extern crate std;
```

Comment:
This breaks the `no-std` build of `floresta-chain`

Reply:
Oh, right... this is needed because `SystemTime` that i use to validate sequence and locktime... Ill figure out a way to validate on without std